### ISSUE TITLE
Browserify compile error. 

### ISSUE THREAD

**Issue by dobromir-hristov** [RyanMullins/angular-hammer#71]
I get a browserify error that says Angular and Hammer cannot be found. I import them right above the angular-hammer import. When I took a closer look, it was searching inside the wrong folders for them: (node_modules/hammerjs/node_modules/hammerjs/hammerjs.js) something like that. So I went into angular-hammer`s package.json and removed angular and hammerjs from the `browser` property and all got fixed... Is this really how it should be acting?

**Comment by dobromir-hristov** [comment]
This is still an issue and I would love if someone could give me an answer.

**Comment by ipavlic** [comment]
No, it's not supposed to be like that. Commit https://github.com/RyanMullins/angular-hammer/commit/ff754eefb189c10b621465df03d0a6a5f423917f contains a fix for Browserify, but it doesn't seem to be included in the main package.

**Comment by RyanMullins** [comment]
Just did, though I can't say I did much in the way of testing. 

It's probably obvious by this point, but I haven't been using this library much recently and I don't see myself having time to get back to it frequently. I'm going to try and find a solution to long-term maintenance this month.
